test/contracts-network/token-locking.js

```diff
@@ -181,6 +181,8 @@ contract("Token Locking", addresses => {
     it("should correctly increment total lock count", async () => {
       await token.approve(tokenLocking.address, usersTokens, { from: userAddress });
       await tokenLocking.deposit(token.address, usersTokens, { from: userAddress });
+      await fundColonyWithTokens(colony, otherToken);
+      await colony.moveFundsBetweenPots(1, 0, 0, 1, 0, 100, otherToken.address);
```

Comment:
Why are these lines needed now and not before?

Reply:
I no longer allow a payout to start if there are no tokens to be paid out.